lib/nodejs/esm-utils.js

```diff
@@ -27,15 +29,17 @@ const formattedImport = async file => {
       throw err;
     }
   }
-  return import(file);
+  return exports.doImport(esmDecorator(file));
 };
 
-exports.requireOrImport = async file => {
+exports.doImport = async file => import(file);
```

Comment:
This is for the testing purposes. `rewiremock` doesn't work in combination of unexistent files and dynamic imports